src/framework/shortcuts/view/shortcutsmodel.cpp

```diff
@@ -57,11 +58,32 @@ QVariant ShortcutsModel::data(const QModelIndex& index, int role) const
         return QString::fromStdString(action.code) + action.title.qTranslated() + action.description.qTranslated()
                + sequencesToNativeText(shortcut.sequences);
     }
+    case RoleSection:
+        return SectionName(shortcut);
+    case RoleSectionValue:
+        return SectionName(shortcut).toString() + sequencesToNativeText(shortcut.sequences);
```

Comment:
In SectionName, you convert from QString to QVariant and here you're converting back. That could be more efficient. Just return QString in SectionName.